aksel.nav.no/website/next.config.js

```diff
@@ -56,17 +56,21 @@ const nextConfig = {
   transpilePackages: ["@navikt/ds-tokens", "react-hotjar"],
   publicRuntimeConfig: {
     NEXT_PUBLIC_TEST: process.env.NEXT_PUBLIC_TEST,
+    UMAMI_TRACKING_ID:
+      process.env.NODE_ENV === "production"
```

Comment:
Not 100% sure here, but as long as one runs "next build" this is set to production. This means that builds for "Aksel.ansatt.nav.no" also gets the production-tag. Is this intended?

Reply:
correct, becomes development for `yarn dev` but production with `yarn build`, maybe not the best to use `NODE_ENV` for this, i'll look for a better existing value we could hopefully use to properly distinguish dev&prod for this before making something from scratch.... 🤔